src/PrestaShopBundle/Resources/config/services/bundle/form/form_type.yml

```
@@ -2183,3 +2183,12 @@ services:
     public: true
     tags:
       - { name: form.type }
+
+  form.type.improve.international.tax.tax_rules_group_type:
+    class: 'PrestaShopBundle\Form\Admin\Improve\International\Tax\TaxRulesGroupType'
```

```suggestion
  PrestaShopBundle\Form\Admin\Improve\International\Tax\TaxRulesGroupType:
```
Same principle for form types, and all other services I won't detail them all you see the idea I think 😉

---

Fixed